### PR TITLE
Fix flickering on “collapse sidebar” link when hovering (?)

### LINKED ISSUE #121: hovering over "Collapse sidebar" causes looping size change
Str, using mouse in Chrome or Firefox:

1 . click "collapse side bar".
2. click on "click on popout sidebar".
3. hover over just above the text "collapse side bar".

Expected:
That it doesn't shimmer by changing size.

Actual:
The "collapse side bar" element starts changing size erratically.

### PR DESCRIPTION
**Pull request by tripu**
Strange as it sounds, I think the bug is related to the grey overlay that both Chrom* and FF display at the bottom left when hovering over a hyperlink, showing the URL in the `href` attribute of the link.

This PR removes the `href` attribute from `a#toc-toggle`, and fakes a proper link by setting the right `cursor` and colour. That solves it for me in both FF and Chrom*.

Is this too much of a hack? Are the side-effects for semantics and a11y too bad?

Fixes #121.

### CODE REVIEW

**Comment by tripu** [comment]
@marcoscaceres, @fantasai, @tanyeah, @plehegar&hellip; anyone cares to review? :)

**Comment by marcoscaceres** [comment]
Although I've looked at this previously, css is not my strong suit 😞

**Review by marcoscaceres — APPROVED** [review]
Lgtm

**Comment by tripu** [comment]
Thanks, @marcoscaceres.

I'm waiting for a second green light before merging, as I still suspect some people may object to this approach.

**Comment by marcoscaceres** [comment]
Was also hoping @fantasai would take a look 👀

**Comment by dontcallmedom** [comment]
-1 from me - at the very least, this makes the said link non-navigable; while this could probably be fixed with a `tabindex=0`, I think this would also need some more ARIA love.

An alternative that I have seen described would be to use a button (which doesn't show the URL pop up) - which I think would be semantically better than using a link-less link.

**Comment by plehegar** [comment]
I wasn't able to reproduce the bug myself on Chrome and FF for the moment.

**Comment by dontcallmedom** [comment]
here is a screencast showing the bug occuring: http://media.w3.org/2017/09/tr-design-122.mkv

**Comment by tripu** [comment]
@dontcallmedom:

Setting a `tabindex` does not seem to solve the navigation problem in my experiments.

Replacing both elements with `button`s is fine by me, but that'd require a bit of redesign; either to disguise the buttons and make them look like hyperlinks, or to adjust the overall design to make it consistent (eg, a button with borders would look ugly inside these overlays with very rounded borders). I'd summon @fantasai here if that's the way to go. (But then again, all this to avoid some occasional flickering that looks more like a browser bug&hellip;?)

**Comment by tripu** [comment]
What shall we do with this PR?

Is it still better than the alternatives (doing nothing, using buttons instead of hyperlinks)?

**Comment by dontcallmedom** [comment]
I don't think it's better (since it reduces accessibility) so I would suggest closing it

**Comment by tripu** [comment]
Okay, @dontcallmedom

**Pull request closed without merging**
